test/integration/crcsuite/crcsuite.go

```diff
@@ -96,11 +96,13 @@ func FeatureContext(s *godog.Suite) {
 			fmt.Println("Obtaining bundle...")
 			bundle, err := DownloadBundle(bundleURL, ".")
 			if err != nil {
-				fmt.Errorf("Failed to obtain CRC bundle, %v\n", err)
+				fmt.Printf("Failed to obtain CRC bundle, %v\n", err)
+				os.Exit(1)
```

Comment:
@cfergeau how about using `logging.Fatalf("Failed to obtain CRC bundle, %v\n", err)` directly it will also exit and provide the details in the logging?

```
$ make integration BUNDLE_LOCATION=/tmp/foo/crc_hyperkit_4.1.14.crcbundle  PULL_SECRET_FILE=/tmp/foo/pull
Log successfully started, logging into: /Users/prkumar/work/github/crc/test/integration/out/test-results/integration_2019-9-26_08-37-14.log
Running integration test in: /Users/prkumar/work/github/crc/test/integration/out/test-run
Working directory set to: /Users/prkumar/work/github/crc/test/integration/out/test-run
Deleted CRC home folder /Users/prkumar/.crc.
The bash instance has been started and will be used for testing.
Obtaining bundle...
Copying bundle from /tmp/foo/crc_hyperkit_4.1.14.crcbundle to /Users/prkumar/work/github/crc/test/integration/out/test-run.
time="2019-09-26T08:37:14+05:30" level=fatal msg="Failed to obtain CRC bundle, open /tmp/foo/crc_hyperkit_4.1.14.crcbundle: no such file or directory\n"
FAIL	github.com/code-ready/crc/test/integration	0.549s
make: *** [integration] Error 1
```

Reply:
crcsuite.go currently has minimal dependencies on packages provided with crc (but logging is light enough that this not really an issue).
If I use logging.Fatalf though, then the rest of the code which uses fmt.Println stands out and would need to be switched to logging too. As this does not bring much, I'd prefer to stick with fmt.Printf for now.